MIGRATE.md

```diff
@@ -150,12 +164,12 @@ module.exports = {
 
 };
 ```
-
+## Changes Reflected after migration
 In summary, we can see the follow changes were made
 1.  The webpack schema for using loaders has changed
     - `loaders` is now `module.rules`
     -  `query` is now `options`
```

Comment:
There are now also other changes (from 2 to 4)
- https://github.com/webpack/webpack-cli/issues/394
- https://github.com/webpack/webpack-cli/issues/395
- https://github.com/webpack/webpack-cli/issues/396